tasks/testing.rake

```diff
@@ -214,6 +214,15 @@ module MinitestSuite
   end
 end
 
+def execute_cmd_with_env(env, cmd)
+  if RUBY_VERSION.to_f > 2
```

Comment:
@Mogztter shouldn't this be `>= 2` to include 2.0 (I'm assuming env support was introduced since 2.0)

If that's true you can safely assume we're targeting 2.0 only, and leaving older releases for `< 2.0` (I'll update the gemspec soon to reflect that).